src/FakeItEasy/Configuration/IAnyCallConfigurationWithVoidReturnType.cs

```diff
@@ -0,0 +1,11 @@
+namespace FakeItEasy.Configuration
+{
+    /// <summary>
+    /// Aggregate of IVoidArgumentValidationConfiguration&lt;T&gt; and IWhereConfiguration&lt;IAnyCallConfigurationWithVoidReturnType&gt;.

```

Comment:
`IVoidArgumentValidationConfiguration` is not generic - strike the `&lt;T&gt;`.

Reply:
@thomaslevesque, did this comment get lost in the excitement?

Reply:
Yes ! I'll fix it

Reply:
Fixed